Fix Ci test fail

**Ci Test : Mustache Lint error :**

RUN  Mustache Lint on format_hero
/home/runner/work/moodle-format_hero/moodle-format_hero/moodle/course/format/hero/templates/course_summary.mustache - WARNING: HTML Validation error, line 16: CSS: “background-color”: Parse Error. (          <div class="course-img course-img-"                                        style='background-color: ;'                >     )
/home/runner/work/moodle-format_hero/moodle-format_hero/moodle/course/format/hero/templates/course_summary.mustache - WARNING: HTML Validation info, line 34: Empty heading. (          <h3></h3>)
/home/runner/work/moodle-format_hero/moodle-format_hero/moodle/course/format/hero/templates/course_summary.mustache - WARNING: HTML Validation info, line 41: Empty heading. (          <h3></h3>)
/home/runner/work/moodle-format_hero/moodle-format_hero/moodle/course/format/hero/templates/course_summary.mustache - WARNING: HTML Validation error, line 43: Bad value “region” for attribute “role” on element “li”. (          <li id="section-0" class="section main clearfix" role="region" aria-label="overview">     )
/home/runner/work/moodle-format_hero/moodle-format_hero/moodle/course/format/hero/templates/course_summary.mustache - WARNING: HTML Validation info, line 52: Empty heading. (          <h2></h2>)
/home/runner/work/moodle-format_hero/moodle-format_hero/moodle/course/format/hero/templates/local/content.mustache - INFO: Template produced no content
/home/runner/work/moodle-format_hero/moodle-format_hero/moodle/course/format/hero/templates/local/content.mustache - OK: Mustache rendered html succesfully
/home/runner/work/moodle-format_hero/moodle-format_hero/moodle/course/format/hero/templates/course_topic.mustache - WARNING: HTML Validation error, line 8: CSS: “background-color”: Parse Error. (>        <div class="section-navigation navigationtitle section-img section-img-"                        style='background-color: ;'        >     )
/home/runner/work/moodle-format_hero/moodle-format_hero/moodle/course/format/hero/templates/course_topic.mustache - WARNING: HTML Validation info, line 25: Empty heading. (          <h2 class=""></h2>)
/home/runner/work/moodle-format_hero/moodle-format_hero/moodle/course/format/hero/templates/progress_bar.mustache - OK: Mustache rendered html succesfully
/home/runner/work/moodle-format_hero/moodle-format_hero/moodle/course/format/hero/templates/course_summary_reverse.mustache - WARNING: HTML Validation info, line 12: Empty heading. (          <h3></h3>)
/home/runner/work/moodle-format_hero/moodle-format_hero/moodle/course/format/hero/templates/course_summary_reverse.mustache - WARNING: HTML Validation info, line 19: Empty heading. (          <h3></h3>)
/home/runner/work/moodle-format_hero/moodle-format_hero/moodle/course/format/hero/templates/course_summary_reverse.mustache - WARNING: HTML Validation error, line 21: Bad value “region” for attribute “role” on element “li”. (          <li id="section-0" class="section main clearfix" role="region" aria-label="overview">     )
/home/runner/work/moodle-format_hero/moodle-format_hero/moodle/course/format/hero/templates/course_summary_reverse.mustache - WARNING: HTML Validation info, line 30: Empty heading. (          <h2></h2>)
/home/runner/work/moodle-format_hero/moodle-format_hero/moodle/course/format/hero/templates/course_summary_reverse.mustache - WARNING: HTML Validation error, line 43: CSS: “background-color”: Parse Error. (          <div class="course-img course-img-"                                        style='background-color: ;'                >     )
Error: Process completed with exit code 1.

**Ci Test : phpdoc checks error :**

 phpdoc checks
course/format/hero/classes/output/courseformat/content.php
    Line 350: Phpdocs for function content::get_section_info has incomplete parameters list
--
course/format/hero/lib.php
    Line 559: Inline phpdocs tag {@link course_edit_form::definition_after_data()} with incorrect contents found. It must match {@link [valid URL] [description (optional)]} or {@see [valid FQSEN] [description (optional)]}
    Line 593: Inline phpdocs tag {@link moodleform::get_data()} with incorrect contents found. It must match {@link [valid URL] [description (optional)]} or {@see [valid FQSEN] [description (optional)]}
    Line 594: Inline phpdocs tag {@link update_course()} with incorrect contents found. It must match {@link [valid URL] [description (optional)]} or {@see [valid FQSEN] [description (optional)]}
    Line 729: Inline phpdocs tag {@link course_can_delete_section()} with incorrect contents found. It must match {@link [valid URL] [description (optional)]} or {@see [valid FQSEN] [description (optional)]}
Error: Process completed with exit code 1.

--- a/classes/output/courseformat/content.php
+++ b/classes/output/courseformat/content.php
@@ -347,7 +347,7 @@ class content extends content_base {
     /**
      * Get section info for UI display.
      *
-     * @param renderer_base $output
+     * @param stdClass $course The course entry from DB
      * @return array $sections array of section names and ids.
      */
     private function get_section_info(stdClass $course): array {

--- a/lib.php
+++ b/lib.php
@@ -556,7 +556,7 @@ class format_hero extends core_courseformat\base {
     /**
      * Adds format options elements to the course/section edit form.
      *
-     * This function is called from {@link course_edit_form::definition_after_data()}.
+     * This function is called from {@see course_edit_form::definition_after_data()}.
      *
      * @param MoodleQuickForm $mform form the elements are added to.
      * @param bool $forsection 'true' if this is a section edit form, 'false' if this is course edit form.
@@ -590,8 +590,8 @@ class format_hero extends core_courseformat\base {
      * In case if course format was changed to 'hero', we try to copy options
      * 'hiddensections' from the previous format.
      *
-     * @param stdClass|array $data return value from {@link moodleform::get_data()} or array with data
-     * @param stdClass $oldcourse if this function is called from {@link update_course()}
+     * @param stdClass|array $data return value from {@see moodleform::get_data()} or array with data
+     * @param stdClass $oldcourse if this function is called from {@see update_course()}
      *     this object contains information about the course before update
      * @return bool whether there were any changes to the options values
      */
@@ -726,7 +726,7 @@ class format_hero extends core_courseformat\base {
     /**
      * Whether this format allows to delete sections.
      *
-     * Do not call this function directly, instead use {@link course_can_delete_section()}
+     * Do not call this function directly, instead use {@see course_can_delete_section()}
      *
      * @param int|stdClass|section_info $section
      * @return bool

--- a/templates/course_summary.mustache
+++ b/templates/course_summary.mustache
@@ -16,6 +16,10 @@
     This template renders the course sumamry section.
     Example context (json):
     {
+        "headerbackcolor": "#FFFFFF",
+        "summaryname": "Summary",
+        "detailsheading": "Details",
+        "resourcesheading": "Resources"
     }
 }}
 
@@ -92,7 +96,7 @@
 
                 <h3>{{resourcesheading}}</h3>
                 <ul class="hero">
-                    <li id="section-0" class="section main clearfix" role="region" aria-label="overview">
+                    <li id="section-0" class="section main clearfix" aria-label="overview">
                         <span class="hidden sectionname">overview</span>
                         <div class = "content">
                             {{#mods}}

--- a/templates/course_summary_reverse.mustache
+++ b/templates/course_summary_reverse.mustache
@@ -16,6 +16,10 @@
     This template renders the course sumamry section.
     Example context (json):
     {
+        "headerbackcolor": "#FFFFFF",
+        "summaryname": "Summary",
+        "detailsheading": "Details",
+        "resourcesheading": "Resources"
     }
 }}
 
@@ -57,7 +61,7 @@
 
                 <h3>{{resourcesheading}}</h3>
                 <ul class="hero">
-                    <li id="section-0" class="section main clearfix" role="region" aria-label="overview">
+                    <li id="section-0" class="section main clearfix" aria-label="overview">
                         <span class="hidden sectionname">overview</span>
                         <div class = "content">
                             {{#mods}}

--- a/templates/course_topic.mustache
+++ b/templates/course_topic.mustache
@@ -21,7 +21,8 @@
 
     Example context (json):
     {
-
+        "headerbackcolor": "#FFFFFF",
+        "sectionname": "Topic"
     }
 }}
 <div class="hero">

